server/src/main/java/io/littlehorse/server/streams/taskqueue/OneTaskQueue.java

```diff
@@ -79,17 +89,16 @@ public void onRequestDisconnected(PollTaskRequestObserver disconnectedObserver)
      *                        that was just
      *                        scheduled.
      */
-    public void onTaskScheduled(ScheduledTaskModel scheduledTaskId) {
+    public boolean onTaskScheduled(ScheduledTaskModel scheduledTaskId) {
```

Comment:
Could you please add a comment here saying what the return type is? I am guessing that it is whether or not we need to do a rehydration, but I don't think it would be obvious to someone looking at this code for the first time.

Reply:
Oops. This is no longer required.